docker-compose.yml

```diff
@@ -26,18 +25,4 @@ services:
       POSTGRES_PASSWORD: d872$$dh
       POSTGRES_USER: itig
 
-  web:
```

Comment:
How will the front end be linked into this?

Reply:
How does that run the front end? The **web:** service linked to the doubtfire-web project and made it possible to use the one docker compose to launch the full application (front end and back end)